### PR TITLE
154769-ElasticSearch-DRUQ-DependencyResolutionException

### PR DESCRIPTION
**Pull request by SQLBob**
154769 enqueue messages for latest hub only
This change was tested in association with [#5790: 154769 ES DependencyResolutionException](https://github.com/daptiv/Ppm/pull/5790#issuecomment-328690906)

- [x] @dachew 

@creg, would you review as well please?

- [x] @creg

### CODE REVIEW

**Comment by dachew** [comment]
:shipit:

**Comment by dachew** [comment]
@SQLBob bump.  What's left to do?

File: Druq.Hub/Queuing/PartitionMaintenance.cs [hunk 1]
```diff
@@ -101,6 +101,7 @@ BEGIN TRANSACTION
     FROM MessagesToProcess m
     INNER JOIN [dbo].[druq_Subscriber] s 
         ON m.[Type] = s.MessageType
+    WHERE IsLatest = 1
```
Comment (creg) [inline Druq.Hub/Queuing/PartitionMaintenance.cs]: Would you please add the table alias here (m.IsLatest or s.IsLatest)?

Comment (creg) [inline Druq.Hub/Queuing/PartitionMaintenance.cs]: Also, inconsistent use of brackets. Would you clean it up?

**Comment by creg** [comment]
A couple of code clarity changes requested, then :shipit:

**Comment by dachew** [comment]
:shipit:

**Pull request merged**